bin/generate_analytics_JSONL_files.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+jar_dir=$CONDA_PREFIX/share/atlas-cli
+
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $scriptDir/common_routines.sh
+
+require_env_var "SOLR_HOST"
+require_env_var "ZK_HOST"
+require_env_var "ZK_PORT"
+require_env_var "BIOENTITIES"
+require_env_var "output_dir"
+require_env_var "EXPERIMENT_FILES"
+require_env_var "jdbc_url"
+require_env_var "jdbc_username"
+require_env_var "jdbc_password"
+require_env_var "server_port"
+
+require_env_var "SPECIES" 
+require_env_var "ACCESSIONS"
+require_env_var "BIN_MAP"
+
+SOLR_PORT=$(get_port_from_hostport $SOLR_HOST)
+SOLR_HOST=$(get_host_from_hostport $SOLR_HOST)
+
+require_env_var "SOLR_PORT"
+
+JAVA_OPTS="-Dsolr.host=$SOLR_HOST"
+JAVA_OPTS="$JAVA_OPTS -Dsolr.port=$SOLR_PORT"
+JAVA_OPTS="$JAVA_OPTS -Dzk.host=$ZK_HOST"
+JAVA_OPTS="$JAVA_OPTS -Dzk.port=$ZK_PORT"
+JAVA_OPTS="$JAVA_OPTS -Ddata.files.location=$BIOENTITIES"
+JAVA_OPTS="$JAVA_OPTS -Dexperiment.files.location=$EXPERIMENT_FILES"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.url=$jdbc_url"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.username=$jdbc_username"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.password=$jdbc_password"
+JAVA_OPTS="$JAVA_OPTS -Dserver.port=$server_port"
+
+# Generate JSONL files from bulk experiments
+
+cmd="java $JAVA_OPTS -jar $jar_dir/atlas-cli-bulk.jar"
```

Comment:
```suggestion
cmd="java -jar $jar_dir/atlas-cli-bulk.jar"
```
I think we can drop it from here then, since the JVM process normally reads this specific var.

Reply:
@pcm32 I have reintroduced `$JAVA_OPTS` as deleting it fails the tests (8b406ba). Apparently, `JAVA_TOOL_OPTIONS` and `_JAVA_OPTIONS` are ways to specify JVM arguments as environment variables instead of command line parameters, and are picked by at least `java` and `javac`. `JAVA_OPTS` is a convention used by Apache Tomcat and some other apps, but it is not read directly by any JDK tools published by Sun/Oracle